### PR TITLE
Task/implement booking layer

### PR DESCRIPTION
**Pull request by slizko1**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -84,19 +84,6 @@
...
-			<version>${mapstruct.version}</version>
-		</dependency>
-		<dependency>
-			<groupId>org.springframework.boot</groupId>
-			<artifactId>spring-boot-starter-security</artifactId>
-		</dependency>
-		<dependency>
-			<groupId>org.hibernate.validator</groupId>
-			<artifactId>hibernate-validator</artifactId>
-		</dependency>
```
[the remark this answers is not in the document]
Comment (VolandevlodD) [reply pom.xml:99]: let`s merge master into this branch and resolve conflicts

File: src/main/java/com/ua/accommodation/dto/booking/BookingResponseDto.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import lombok.Data;
+
+@Data
+public class BookingResponseDto {
+    private Long id;
+    private LocalDate checkInDate;
+    private LocalDate checkOutDate;
+    private Long accommodationID;
+    private Long userId;
+    private Booking.Status status;
```
Comment (vikkushnir) [inline src/main/java/com/ua/accommodation/dto/booking/BookingResponseDto.java:14]: Use import static Booking to write only Status

**Review by vikkushnir — CHANGES_REQUESTED** [review]
Looks good! Only fix a few suggestions

**Review by VolandevlodD — APPROVED** [review]
lgtm pls pay attention to comments

File: src/main/java/com/ua/accommodation/exception/CustomGlobalExceptionHandler.java [hunk 3]
```diff
@@ -1,21 +1,77 @@
...
+
+    @ExceptionHandler(AccessDeniedException.class)
+    protected ResponseEntity<Object> handleUserNotFoundException(AccessDeniedException ex) {
+        String bodyOfResponse = "Access denied: " + ex.getMessage();
+        log.error("AccessDeniedException occurred", ex);
+        return getResponseEntity(HttpStatus.FORBIDDEN, bodyOfResponse);
+    }
+
+    @ExceptionHandler(AccommodationUnavailableException.class)
+    protected ResponseEntity<Object> handleUserNotFoundException(
```
Comment (TonyH277) [inline src/main/java/com/ua/accommodation/exception/CustomGlobalExceptionHandler.java:71]:
```suggestion
    protected ResponseEntity<Object> handleAccommodationUnavailableExceptio(
```

**Review by TonyH277 — APPROVED** [review]
Nice!

**Review by vikkushnir — APPROVED** [review]

**Pull request merged**